src/components/TabPanel/index.tsx

```diff
@@ -79,18 +85,55 @@ export class TabPanel extends React.Component<TabPanelProps> {
         const navCx = 'cr-tab-panel__navigation-container-navigation';
 
         return (
-            <div className={className}>
-                <div className="cr-tab-panel__navigation-container draggable-container">
-                    <div className={navCx} role="tablist">
-                        {panels.map(this.renderTabPanel)}
+            <React.Fragment>
+                {!dropdown ? (
```

Comment:
1. Need to check for truth (!dropdown -> dropdown)
2. Next code is unnecessary duplicated. Need to render conditionally only "..." part. Can be defined additional method for this.
```
<div className={className}>
    <div className="cr-tab-panel__navigation-container draggable-container">
          ...
          {optionalHead && <div className="cr-tab-panel__optinal-head">{optionalHead}</div>}
     </div>
     {contents}
</div>